bin/GameIndex.dbf

```diff
@@ -15343,6 +15366,11 @@ Region = PAL-M5
 Serial = SLES-54021
 Name   = Ruff Trigger
 Region = PAL-M5
+EETimingHack = 1
+[patches = b02c81e5]
+	// Fixes random hangs when going ingame.
+	patch=1,EE,001128e0,word,00000000
+[/Patches]
```

Comment:
I prefer [patches], [/patches] not to have capital letters, check the other changes below as well.